src/maui/SSW.Rewards.Mobile/SSW.Rewards.Mobile/Controls/TitleView.xaml.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System.Windows.Input;
+using CommunityToolkit.Mvvm.Messaging;
+using SSW.Rewards.Mobile.Messages;
+
+namespace SSW.Rewards.Mobile.Controls
+{
+    [XamlCompilation(XamlCompilationOptions.Compile)]
+    public partial class TitleView : ContentView
+    {
+        public static readonly BindableProperty TitleProperty = BindableProperty.Create(
+                                                          propertyName: "TitleText",
+                                                          returnType: typeof(string),
+                                                          declaringType: typeof(TitleView),
+                                                          defaultValue: "",
+                                                          defaultBindingMode: BindingMode.TwoWay,
+                                                          propertyChanged: TitlePropertyChanged);
+
+        public static readonly BindableProperty TapTitleCommandProperty = BindableProperty.Create("TapTitleCommand", typeof(ICommand), typeof(TitleView), null);
+
+        public string Title
+        {
+            get { return base.GetValue(TitleProperty).ToString(); }
+            set { base.SetValue(TitleProperty, value); }
+        }
+
+        public ICommand TapTitleCommand
+        {
+            get
+            {
+                return (ICommand)GetValue(TapTitleCommandProperty);
+            }
+            set
+            {
+                SetValue(TapTitleCommandProperty, value);
+            }
+        }
+
+        private static void TitlePropertyChanged(BindableObject bindable, object oldValue, object newValue)
+        {
+            var control = (TitleView)bindable;
+            control.title.Text = newValue.ToString();
+        }
+
+        public TitleView()
+        {
+            InitializeComponent();
+        }
+
+        private void TapGestureRecognizer_Tapped(object sender, System.EventArgs e)
+        {
+            //App.Current.MainPage.DisplayAlert("Title bar tapped", "I'd tap that", "OK");
+            WeakReferenceMessenger.Default.Send(new ScrollToTopMessage());
```

Comment:
At that moment no one receives `ScrollToTopMessage`. Should we still keet it?

Reply:
Adam want's this functionality and has asked that we conduct a user survey to determine what percentage of users expect this functionality before we remove it.